README.md

```diff
@@ -80,22 +96,29 @@ aws_region = <YOUR_AWS_RDS_REGION, e.g., us-west-2>
 aws_access_key_id = <YOUR_AWS_ACCESS_KEY_ID>
 aws_secret_access_key = <YOUR_AWS_SECRET_ACCESS_KEY>
 
-# You can optionally add more servers by adding more sections similar to the above
+# You can optionally add more servers by adding more sections similar to the above, but with parameters for Google Cloud SQL
 # [server2]
-# db_host = <YOUR_PG_DATABASE_HOST, e.g., xyz.abcdefgh.us-west-2.rds.amazonaws.com>
+# db_host = <YOUR_PG_DATABASE_HOST, e.g., localhost>
 # db_name = <YOUR_PG_DATABASE_NAMES, e.g., postgres>
 # db_username = <YOUR_PG_DATABASE_USER_NAME, e.g., postgres>
 # db_password = <YOUR_PG_DATABASE_PASSWORD>
 # db_port = <YOUR_PG_DATABASE_PASSWORD, e.g., 5432>
-# aws_db_instance_id = <YOUR_AWS_RDS_INSTANCE_ID, e.g., xyz>
-# aws_region = <YOUR_AWS_RDS_REGION, e.g., us-west-2>
-# aws_access_key_id = <YOUR_AWS_ACCESS_KEY_ID>
-# aws_secret_access_key = <YOUR_AWS_SECRET_ACCESS_KEY>
+# gcp_project_id = <YOUR_GCP_PROJECT_ID>
+# gcp_cloudsql_instance_id = <YOUR_GCP_CLOUDSQL_INSTANCE_ID>
+# gcp_credentials_file = <YOUR_GCP_CREDENTIALS_FILE (default value: ~/.config/gcloud/application_default_credentials.json)>
+EOF
 ```
 
-PostgreSQL connection strings are of the form `postgres://<db_username>:<db_password>@<db_host>:<db_port>/<db_name>`.
+### Note: 
+ - PostgreSQL connection strings are of the form `postgres://<db_username>:<db_password>@<db_host>:<db_port>/<db_name>`.
+
+  - If you're using AWS RDS, then your `<db_host>` is in this format: `<aws_db_instance_id>.<aws_account_id>.<aws_region>.rds.amazonaws.com`
 
-If you're using AWS RDS, then your `<db_host>` is in this format: `<aws_db_instance_id>.<aws_account_id>.<aws_region>.rds.amazonaws.com`
+  - For Google Cloud SQL, you need to follow [these instructions](https://cloud.google.com/sql/docs/postgres/connect-auth-proxy) to install gcloud CLI and cloud-sql-proxy (if your database is not directly accessible from this machine). Then, you need to:
+    - init `gcloud`: `gcloud init`
+    - enable default auth on  `gcloud`: `gcloud auth application-default login`
```

Comment:
I think  we are missing `gcloud auth login` as a prerequisite step before auth application-default login?

Reply:
Thanks. Added it.